recipes/fltk/all/conanfile.py

```diff
@@ -154,7 +162,8 @@ def package_info(self):
                 self.cpp_info.system_libs.append("gdiplus")
             if self.options.with_gl:
                 self.cpp_info.system_libs.append("opengl32")
-
+            if Version(self.version) >= "1.4.0":
+                self.cpp_info.system_libs.append("ws2_32")
```

Comment:
Hello @toge, that variable is only being used in the case of Win32 [here](https://github.com/fltk/fltk/compare/release-1.3.9...release-1.4.0-1#diff-9b7279790d300a16b871412df0664aab656e9942969cdf51d3a10994d0eac1a7R43) could you help me better understand this? 😄

Reply:
`ws2_32` is added to `FLTK_LDLIBS`. [link](https://github.com/fltk/fltk/blob/release-1.4.0/CMake/variables.cmake#L43)
`FLTK_LDLIBS` is used to generate `LIBS` variables. [link](https://github.com/fltk/fltk/blob/release-1.4.0/CMake/variables.cmake#L129)
`LIBS` variable is used in [fltk-config.in](https://github.com/fltk/fltk/blob/release-1.4.0/fltk-config.in#L39).

In hand install situation, `fltk-config` is used to compile and link user program. (ex. g++ `fltk-config --libs` xxx.cpp)